src/app/loadout/Loadouts.tsx

```diff
@@ -60,7 +64,12 @@ export default function LoadoutsContainer({ account }: { account: DestinyAccount
     return <ShowPageLoading message={t('Loading.Profile')} />;
   }
 
-  return <Loadouts account={account} />;
+  // TODO: how high in our tree do we want this analyzer?
```

Comment:
This seems fine, though if you moved it down into `Loadouts` I bet you could directly assign `selectedStoreId` as a prop rather than having to use `useUpdateLoadoutAnalysisContext`.

Reply:
I actually did that first but ran into the issue that the `useSummaryLoadoutsAnalysis` hook also needs to exist in Loadouts and that one needs the context outside of it.

There's also a case to be made for placing it even higher in the tree (around our routes) since right now navigating away from the Loadouts page cleans up the analyzer and discards all results.